app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from typing import Dict
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[Dict], cafe: Cafe) -> str:
```

Comment:
The type hint `list[Dict]` is not valid in Python versions below 3.9. If you are using a version below 3.9, consider using `List[Dict]` from the `typing` module instead.